addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py

```diff
@@ -308,15 +309,15 @@ def _find_value(xpath, element=tree):
 
         invoice_date_node = tree.find('./{*}ExchangedDocument/{*}IssueDateTime/{*}DateTimeString')
         if invoice_date_node is not None and invoice_date_node.text:
-            date_str = invoice_date_node.text
+            date_str = re.sub("[^0-9]", "", invoice_date_node.text)
             date_obj = datetime.strptime(date_str, DEFAULT_FACTURX_DATE_FORMAT)
             invoice_form.invoice_date = date_obj.strftime(DEFAULT_SERVER_DATE_FORMAT)
 
         # ==== invoice_date_due ====
 
         invoice_date_due_node = tree.find('.//{*}SpecifiedTradePaymentTerms/{*}DueDateDateTime/{*}DateTimeString')
         if invoice_date_due_node is not None and invoice_date_due_node.text:
-            date_str = invoice_date_due_node.text
+            date_str = re.sub("[^0-9]", "", invoice_date_due_node.text)
```

Comment:
I'd put this patter to a new const and define it next to the definition of `DEFAULT_FACTURX_DATE_FORMAT`

Reply:
Changes are done!